recipes/seroba/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "seroba" %}
+{% set version = "1.0.0" %}
+{% set md5sum = "fcd6802070aa90d3f4810bb72a23cdce" %}
```

Comment:
`sha256` [should be preferred](http://bioconda.github.io/guidelines.html#hashes) unless only `md5` is provides alongside the source.